### PR TITLE
Added L0 tests

### PR DESCRIPTION
**Pull request by sundargs2000**

### CODE REVIEW

File: node_modules/.bin/tsc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
 
 case `uname` in
-    *CYGWIN*) basedir=`cygpath -w "$basedir"`;;
+    *CYGWIN*|*MINGW*|*MSYS*) basedir=`cygpath -w "$basedir"`;;
```
Comment (ajinkya599) [inline node_modules/.bin/tsc:5]: Remove the changes in dev dependencies?
Run `npm prune --production`

File: __tests__/utilities.test.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    });
+
+    test('getExecutableExtension() - return empty string for non-windows OS', async () => {
+        mockStatusCode = 1;
+        stdOutMessage = ''; 
+        stdErrMessage = 'Directory doesn\'t exist'; 
+        await expect(utils.execCommand('cd', ['nonEsistingDirectory'], {} as ExecOptions)).rejects.toThrow('Directory doesn\'t exist');         
+    });
+
+    test('getExecutableExtension() - return empty string for non-windows OS', async () => {
```
Comment (shigupt202) [inline __tests__/utilities.test.ts:57]: These statements are same for multiple tests. Please update them as per the tests and placeholder in describe as well.

File: __tests__/utilities.test.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    return {
+        ToolRunner: jest.fn().mockImplementation((toolPath, args, options) => {
+            return {
+                exec: () => mockExecFn(toolPath, args, options)  
+            }
+        })
+    }
+});
+
+describe('This is a placeholder for intial test cases, to be removed', () => {
```
Comment (shigupt202) [inline __tests__/utilities.test.ts:21]: Update this as well

**Review by shigupt202 — APPROVED** [review]

**Comment by shigupt202** [comment]
@sundargs2000 What are the new commits for?

**Comment by sundargs2000** [comment]
> @sundargs2000 What are the new commits for?

Hey. There some changes in node_modules so Ajinkya suggested to remove them.

File: lib/helm-util.js [hunk 4]
```diff
@@ -10,22 +10,15 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
...
-};
 Object.defineProperty(exports, "__esModule", { value: true });
-const os = __importStar(require("os"));
-const path = __importStar(require("path"));
-const util = __importStar(require("util"));
-const fs = __importStar(require("fs"));
+const os = require("os");
+const path = require("path");
+const util = require("util");
+const fs = require("fs");
```
Comment (shigupt202) [inline lib/helm-util.js:17]: Were these changes auto generated on build? Also, have you tested all the final changes? If everything's done, I'll merge the PR.
CC: @ajinkya599

Comment (sundargs2000) [reply lib/helm-util.js:17]: yes they were auto generated. ive tested it. But baking with kompose was failing for some reason (even using azure/k8s-bake@v1), need to look at that. Ajinkya can have a look at the L0 part of this, I still need to change repo [here](https://github.com/sundargs2000/k8s-bake/blob/7fc7fc11b1124768cffd7c7f580fbe4e9010ac5e/.github/workflows/TriggerIntegrationTests.sh#L25) to Azure. Jyotsna now has write access to this repo so she can merge after that.

**Pull request merged**